### DISCUSSION TITLE
Notifications suddenly massive

### DISCUSSION THREAD

**Discussion by Kwaddarth** [prasanthrangan/hyprdots#2041] [Q&A]
my notifications suddenly started to become weirdly large, also wondering how to disable notifications for spotify as it displays a new notification every time the song changes.

![image](https://github.com/user-attachments/assets/8af121a5-7cd8-40ff-9048-1edbaf4401b4)

**Comment by richen604** [comment]
refer to https://github.com/prasanthrangan/hyprdots/pull/2038#issue-2726535993

**Comment by Supertocho80** [comment]
To quit notifications in Spotify:
https://community.spotify.com/t5/Desktop-Linux/Desktop-Linux-sends-a-notification-each-time-a-song-starts/td-p/5392871
